torchvision/ops/misc.py

```diff
@@ -124,12 +124,13 @@ class FrozenBatchNorm2d(torch.nn.Module):
     are fixed
     """
 
-    def __init__(self, n):
+    def __init__(self, num_features, eps=0.):
```

Comment:
for backwards-compatibility, can you also define a `n=None` kwarg?
Something like
```python
def __init__(self, num_features, eps=0.0, n=None):
    if n is not None:
        warnings.warn("n is deprecated and has been renamed num_features")
        num_features = n
```

Reply:
Just added it, thanks for the suggestion!